src/store/Workspaces/index.ts

```diff
@@ -234,6 +235,19 @@ export const actionCreators: ActionCreators = {
 
   },
 
+  requestWorkspace: (workspaceId: string): AppThunk<KnownAction, Promise<void>> => async (dispatch): Promise<void> => {
+    dispatch({ type: 'REQUEST_WORKSPACES' });
+
+    try {
+      const workspace = await WorkspaceClient.restApiClient.getById<che.Workspace>(workspaceId);
+      dispatch({ type: 'UPDATE_WORKSPACE', workspace });
+    } catch (e) {
+      dispatch({ type: 'RECEIVE_ERROR' });
+      const message = e.response && e.response.data && e.response.data.message ? e.response.data.message : e.message;
+      throw new Error(`Failed to update. ${message}`);
```

Comment:
Why don't you throw the error in the way it's already done in this module? What is the benefit of this?

Reply:
This is due to the transition to use restApiClient.  It is needed for showing a real message instead of ```...[object][object].```